st/clitests/jclient_spec.py

```diff
@@ -361,6 +361,121 @@ def get_response_elements(response):
 
     JClientTest('Jclient can download 700K file uploaded in chunked mode').get_object("seagatebucket", "700Kfilec").execute_test().command_is_successful().command_created_file("700Kfilec")
 
+    # ************ 8K Unaligned FILE TEST ************
+    JClientTest('Jclient can upload 8K Unaligned file') \
+    .put_object("seagatebucket", "8KUnalignfile", 8400) \
+    .execute_test() \
+    .command_is_successful()
+
+    JClientTest('Jclient can upload 8K Unaligned file in chunked mode') \
+    .put_object("seagatebucket", "8KUnalignfilec", 8400, chunked=True) \
+    .execute_test() \
+    .command_is_successful()
+
+    JClientTest('Jclient can download 8K Unaligned file') \
+    .get_object("seagatebucket", "8KUnalignfile") \
+    .execute_test() \
+    .command_is_successful() \
+    .command_created_file("8KUnalignfile")
+
+    JClientTest('Jclient can download 8K Unaligned file uploaded in chunked mode') \
+    .get_object("seagatebucket", "8KUnalignfilec") \
+    .execute_test() \
+    .command_is_successful() \
+    .command_created_file("8KUnalignfilec")
+
+    # ************ 16K FILE TEST ************
+    JClientTest('Jclient can upload 16K file') \
+    .put_object("seagatebucket", "16Kfile", 16384) \
+    .execute_test() \
+    .command_is_successful()
+    
+    JClientTest('Jclient can upload 16K file in chunked mode') \
+    .put_object("seagatebucket", "16Kfilec", 16384, chunked=True) \
+    .execute_test() \
+    .command_is_successful()
+
+    JClientTest('Jclient can download 16k file') \
+    .get_object("seagatebucket", "16Kfile") \
+    .execute_test() \
+    .command_is_successful() \
+    .command_created_file("16Kfile")
+
+    JClientTest('Jclient can download 16k file uploaded in chunked mode') \
+    .get_object("seagatebucket", "16Kfilec") \
+    .execute_test() \
+    .command_is_successful() \
+    .command_created_file("16Kfilec")
+
+    # ************ 16K Unaligned FILE TEST ************
+    JClientTest('Jclient can upload 16K Unaligned file') \
+    .put_object("seagatebucket", "16KUnalignfile", 16500) \
+    .execute_test() \
+    .command_is_successful()
+
+    JClientTest('Jclient can upload 16K Unaligned file in chunked mode') \
+    .put_object("seagatebucket", "16KUnalignedfilec", 16500, chunked=True) \
+    .execute_test() \
+    .command_is_successful()
+
+    JClientTest('Jclient can download 16k Unaligned file') \
+    .get_object("seagatebucket", "16KUnalignedfile") \
+    .execute_test() \
+    .command_is_successful() \
+    .command_created_file("16KUnalignedfile")
+
+    JClientTest('Jclient can download 16k Unaligned file uploaded in chunked mode') \
+    .get_object("seagatebucket", "16KUnalignedfilec") \
+    .execute_test() \
+    .command_is_successful() \
+    .command_created_file("16KUnalignedfilec")
+
+    # ************ 1M FILE TEST ************
+    JClientTest('Jclient can upload 1M file') \
```

Comment:
In case if we are uploading 1M/16k.... files using s3cmd/aws then via jclient not needed. If not then keep it
jclient test with chunk (of different sizes) are important.